weave-js/src/urls.ts

```diff
@@ -0,0 +1,55 @@
+export const URL_RECENT = 'recent';
+export const URL_WANDB = 'wandb';
+export const URL_DRAFTS = 'drafts';
+
+type AssetType = 'boards' | 'tables';
```

Comment:
Something to flag: `boards` and `tables` are sort of special cases (there is business logic to define these as special citizens). In the near-term we also will expose-user-defined types. I just want to ensure we don't have conflict here. fwiw i see a path forward for something like `entity/project/weave_object/[custom_type]/`

Reply:
I could make this a string instead of an enum to make it more flexible. Would we let people create custom types with the same name as our built-ins? Wondering if we need a reserved prefix or something similar.

Reply:
In this case `boards` and `tables` are just application-level logic, not really reserved words. In the sense that we could easily decide to call `boards` something like `dash` or `pages` or whatever. They just map to `Group` panels with a certain config. Similar to `tables`. It might be the case that `tables` could be `data` or something. `tables` maps to two different types: `run_table` and `stream_table`. So anyhow, i think the enum is fine, it is more that this path components `assetType` is defined by us as app devs, but it is perfectly reasonable in the near future to have types of things ("weave objects") that are user-defined type names